docs/sources/panels/field-configuration-options.md

```diff
@@ -61,7 +61,7 @@ Custom field options are:
 
 ## Configure all fields
 
-To change how all fields display data, you apply a [field option](#field-options). Usually you apply changes that you want to most or all of the fields here, than apply field overrides to exceptions.
+To change how all fields display data, you apply a [field option](#field-options). Usually you apply changes that you want, to most or all of the fields here, than apply field overrides to exceptions.
```

Comment:
```suggestion
To change how all fields display data, you apply a [field option](#field-options). Usually you apply changes that you want to most of or all of the fields here, rather than applying field overrides to exceptions.
```